bazel_tools/scala.bzl

```diff
@@ -173,11 +176,24 @@ def _set_jvm_flags(
     })
     return result
 
-def _wrap_rule(rule, name = "", scalacopts = [], plugins = [], generated_srcs = [], **kwargs):
+def _wrap_rule(
+        rule,
+        name = "",
+        scalacopts = [],
+        plugins = [],
+        generated_srcs = [],  # hiding from the underlying rule
```

Comment:
My `hiding` you mean we don’t pass it along? Do you know why we even have this field then?